bot/constants.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from os import environ
+from typing import NamedTuple
+
+log = logging.getLogger(__name__)
+
+SIR_ROBIN_PREFIX = "&"
```

Comment:
This should be removed in favour of `Client.prefix`.

Reply:
[a8fc686](https://github.com/python-discord/sir-robin/pull/10/commits/a8fc686e88045f580c583045557690e7611d7e08)